NEWS.md

```diff
@@ -1,6 +1,13 @@
 This is a log of major changes in Gadfly between releases. It is not exhaustive.
 Each release typically has a number of minor bug fixes beyond what is listed here.
 
+# Version 1.3.3
+
+ * Add support for CategoricalArrays 0.10 (#1537)
+ * Add support for Showoff 1.0 (#1522)
+ * Make Gadfly "relocatable" by not loading files at runtime (#1530)
+ * Disable CI testing for Julia < 1.4 (#1525)
```

Comment:
The issue numbers should be in reverse order.  Also the main idea in NEWS.md was to document grammar of graphics features, not housekeeping issues.

Reply:
Good point. My bad. I've reversed the order and only kept the version bumps. That's a trade-off between documenting grammar of graphics features and not having an empty NEWS entry, which, I guess, could be a bit confusing for the reader.